src/device.rs

```diff
@@ -1811,7 +1811,7 @@ impl Runtime {
             let derp_config = DerpConfig {
                 secret_key,
                 servers: SortedServers::new(config.derp_servers.clone().unwrap_or_default()),
-                allowed_pk: peers,
+                meshnet_peers: peers,
```

Comment:
This is probably out of scope for this PR and even more probably not significant, but you can optimize the creation of this `peers` set by doing this:
```
let peers: HashSet<PublicKey> = config
            .as_ref()
            .map(|c| {
                c.peers
                    .as_ref()
                    .map(|peers| peers.iter().map(|peer| peer.base.public_key).collect())
            })
            .flatten()
            .unwrap_or_default();
```
It would prevent a relatively expensive clone, but it could also turn out to have no benefit whatsoever